freckle-app/package.yaml

```diff
@@ -167,6 +168,7 @@ tests:
       - mtl
       - nonempty-containers
       - postgresql-simple
+      - servant-server
```

Comment:
To avoid piling up dependencies it might make sense to make multiple `TextualEnum`-like newtypes in different packages and then use them like

```haskell
newtype Foo = Foo Text
  deriving Csv.ToField via CsvTextualEnum Foo
  deriving Servant.Whatever via ServantTextualEnum Foo
  -- etc.
```

The textual enum concept is something that basically ever component of our software is going to want, so it would be nice if that didn't always have to automatically incur a dependency on myriad packages (cassava, persistent, servant, autodocodec, quickcheck, ...) that cover a vast variety of unrelated domains.

Reply:
Maybe a good thing to do eventually, for sure.

Reply:
@chris-martin, @joelmccracken  last commit has spec use deriving-via instead of newtype.